<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMBZ-G CharLoader tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>5</y>
      <width>1009</width>
      <height>713</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_general">
     <attribute name="title">
      <string>General</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_anims">
     <attribute name="title">
      <string>Animations</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>4</y>
        <width>1001</width>
        <height>681</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <property name="columnCount">
            <number>1</number>
           </property>
           <column>
            <property name="text">
             <string>Animations</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>Idle</string>
            </property>
            <item>
             <property name="text">
              <string>Action 1</string>
             </property>
             <item>
              <property name="text">
               <string>frame</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>delay</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Add...</string>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
               </font>
              </property>
             </item>
            </item>
            <item>
             <property name="text">
              <string>Action 2</string>
             </property>
             <item>
              <property name="text">
               <string>frame</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>delay</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Add...</string>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
                <kerning>true</kerning>
               </font>
              </property>
             </item>
            </item>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>frame</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>delay</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_effects">
     <attribute name="title">
      <string>Effects</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_cmdlist">
     <attribute name="title">
      <string>Command List</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
